plugins/main/public/components/overview/mitre/framework/index.ts

```diff
@@ -10,4 +10,5 @@
  * Find more information about this on the LICENSE file.
  */
 
+//export { Mitre, ITactic } from './mitre';
```

Comment:
Maybe this can be deleted

Reply:
Solved in da292a701c672ad91a2fb285a1a72fbfbaa178a7